src/Worker.Extensions.DurableTask/DurableTaskClientExtensions.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Net;
+using System.Threading;
+using Azure.Core.Serialization;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.DurableTask.Client;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.Functions.Worker;
+
+/// <summary>
+/// Extensions for <see cref="DurableTaskClient"/>
+/// </summary>
+public static class DurableTaskClientExtensions
+{
+    /// <summary>
+    /// Creates an HTTP response that is useful for checking the status of the specified instance.
+    /// </summary>
+    /// <param name="client">The <see cref="DurableTaskClient"/>.</param>
+    /// <param name="request">The HTTP request that this response is for.</param>
+    /// <param name="instanceId">The ID of the orchestration instance to check.</param>
+    /// <param name="cancellation">The cancellation token.</param>
+    /// <returns>An HTTP 202 response with a Location header and a payload containing instance control URLs.</returns>
+    public static HttpResponseData CreateCheckStatusResponse(
+        this DurableTaskClient client,
+        HttpRequestData request,
+        string instanceId,
+        CancellationToken cancellation = default)
+        => client.CreateCheckStatusResponse(request, instanceId, HttpStatusCode.Accepted, cancellation);
+
+    /// <summary>
+    /// Creates an HTTP response that is useful for checking the status of the specified instance.
+    /// </summary>
+    /// <param name="client">The <see cref="DurableTaskClient"/>.</param>
+    /// <param name="request">The HTTP request that this response is for.</param>
+    /// <param name="instanceId">The ID of the orchestration instance to check.</param>
+    /// <param name="statusCode">The status code.</param>
+    /// <param name="cancellation">The cancellation token.</param>
+    /// <returns>An HTTP response with a Location header and a payload containing instance control URLs.</returns>
+    public static HttpResponseData CreateCheckStatusResponse(
+        this DurableTaskClient client,
+        HttpRequestData request,
+        string instanceId,
+        HttpStatusCode statusCode,
+        CancellationToken cancellation = default)
```

Comment:
Seems a little odd to pass a `CancellationToken` to a synchronous method that doesn't do any I/O, no?

Reply:
It does do I/O though! It writes to the request stream. I took it in because the `ObjectSerializer.Serialize` method I use accepts it.

Reply:
Hmm, yeah, I guess so. It's weird that it's in synchronous code instead of async code, but I suppose that's because HTTP triggers in Azure Functions buffer the request and response streams? Anyways, doesn't hurt to have this, I guess, and maybe makes it more future-proof.